pkgs/servers/knxd/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, stdenv, fetchFromGitHub, buildPackages
+, autoreconfHook, pkg-config, indent, perl
+, argp-standalone, fmt, libev
+, withSystemd ? stdenv.isLinux, systemd
+, withUsb ? stdenv.isLinux, libusb1
+}:
+
+stdenv.mkDerivation rec {
+  pname = "knxd";
+  version = "0.14.59";
+
+  src = fetchFromGitHub {
+    owner = "knxd";
+    repo = "knxd";
+    rev = version;
+    hash = "sha256-m3119aD23XTViQJ2s7hwnJZ1ct4bcEFWuyUQajmqySQ=";
+  };
+
+  postPatch = ''
+    sed -i '2i echo ${version}; exit' tools/version.sh
+    sed -i '2i exit' tools/get_libfmt
+  '';
+
+  depsBuildBuild = [ buildPackages.stdenv.cc ];
+
```

Comment:
This is not the default?

Reply:
Hm, I was adding it to fix some problem with cross ccompilation, but now I've checked it's OK without it. Well, removed.